### PR TITLE
Data generator

### PR DESCRIPTION
**Pull request by CSellers720**
## Converted seed script to generic data generator
This script can now be called to generate data in a non-DBMS-specific manner.
The functions herein will be used to populate both of my databases.

### CODE REVIEW

File: database/genRecords.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  for (let i = 0; i < RECORDS_QUANTITY; i++) {
+    const numberOfResevervations = Math.floor(Math.random() * 25);
+    const adults = Math.ceil(Math.random() * 10);
+    const checkIn = faker.date.soon(90);
+    if (i % Math.floor(RECORDS_QUANTITY / 8) === 0) {
+      progressUpdate(i);
+    }
+    for (let j = 0; j < numberOfResevervations; j++) {
+
+
```
Comment (azizbouland) [inline database/genRecords.js:49]: you have an extra new line

File: database/genRecords.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      trips.push(trip);
+    }
+  }
+  return trips;
+};
+
+const progressUpdate = (i) => {
+  const percentage = (i / RECORDS_QUANTITY) * 100;
+  console.log(`${percentage}% completed.`);
+}
```
Comment (azizbouland) [inline database/genRecords.js:67]: Clever way to capture progress

File: database/index.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ let locationSchema = mongoose.Schema({
 let Trip = mongoose.model('Trip', tripSchema);
 let Location = mongoose.model('Location', locationSchema);
```
Comment (azizbouland) [inline database/index.js:30]: I'm surprised you are using mongoose with Postgres, how so? Are you using mongoDB as well?

File: database/index.js [hunk 4]
```diff
@@ -86,9 +86,19 @@ var getLocationInformation = (locationId) => {
...
-module.exports.getLocationInformation = getLocationInformation;
\ No newline at end of file
+module.exports = {
+  save,
+  getReservationsForLocation,
+  createLocation,
+  getLocationInformation
+  updateTrip,
+  deleteTrip
+}
```
Comment (azizbouland) [inline database/index.js:104]: missing new line at the end

**Review by azizbouland — APPROVED** [review]
Very clean and to the point, nicely done!

File: database/index.js (hunk 3 above)
Comment (CSellers720) [reply database/index.js:30]: I'm not! This is inherited code and it will be pulled out.

File: database/genRecords.js (hunk 1 above)
Comment (CSellers720) [reply database/genRecords.js:49]: Whoops!

File: database/index.js (hunk 4 above)
Comment (CSellers720) [reply database/index.js:104]: Good catch!

**Pull request closed without merging**